### ISSUE TITLE
MapConfig changes should be ignored for existing maps

### ISSUE THREAD

**Issue by puzpuzpuz** [hazelcast/hazelcast#16981]
As shown in https://github.com/hazelcast/management-center/issues/3214, it's possible to programmatically change `MapConfig` for an already created map described via xml config. Once the change is done, it may be seen in the effective XML config available in MC:
https://docs.hazelcast.org/docs/management-center/4.0.2/manual/html/index.html#monitoring-members

Such changes should be ignored or an exception should be thrown.

**Comment by jgardiner68** [comment]
Reported by customer LCH in support issue https://hazelcast.zendesk.com/agent/tickets/6221

**Comment by blazember** [comment]
Duplicate of https://github.com/hazelcast/hazelcast/issues/10812. Closing this in favor of the linked issue that contains some discussion on this topic.

If you think it's not a duplicate for any reason, please reopen.
